File: TransCoder_tokenized_test_set_functions/java/ROTATE_MATRIX_180_DEGREE.java
static void rotateMatrix ( int mat [ ] [ ] ) {
  for ( int i = N - 1;
  i >= 0;
  i -- ) {
    for ( int j = N - 1;
    j >= 0;
    j -- ) System . out . print ( mat [ i ] [ j ] + " " );
    System . out . println ( );
  }
}

File: TransCoder_tokenized_test_set_functions/python/ROTATE_MATRIX_180_DEGREE.py
def rotateMatrix ( mat ) :
    i = N - 1;
    
    while ( i >= 0 ) :
        j = N - 1;
        
        while ( j >= 0 ) :
            print ( mat [ i ] [ j ], end = " " );
            
            j = j - 1;
            
        print ( );
        
        i = i - 1;
        
    
